<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="ribbon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>141</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolBar {
    padding: 0px;
}

QTabWidget:pane {
	border-radius: 0px;
	margin:0px;
	padding: 0px;
    background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #efefef, stop: 0.5 #e5e5e5, stop: 1.0 #fdfdfd);
    border-bottom: 1px solid rgba(0,0,0,20%);
    border-top: 1px solid rgba(0,0,0,20%);
    top: -1;
}
QTabWidget{

}

QTabWidget::tab-bar {
    left: 2px;
}

QTabBar::tab {
    padding-top: 4px;
    padding-bottom: 4px;
    padding-left: 12px;
    padding-right: 12px;
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    margin-right:5px;
    color:#2A2A2A;
 }

QTabBar::tab:hover {
    background: rgba(255, 119, 0, 20%);
}

QTabBar::tab:selected{
    background: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #fdfdfd, stop: 1.0 #efefef);
    color: #2A2A2A;

    border: 1px solid rgba(0,0,0,20%);
    border-bottom-color: #f9f9f9;
}
QToolButton {
    text-align: left;
    border: 1px solid transparent;
    padding: 0px;
    margin-left:2px;
    margin-right:2px;
}

QToolButton:hover {
    border: 1px solid rgba(120,160,255,60%);
    background-color: QLinearGradient(spread:pad, x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgba(150,180,200,30%),stop: 0.5 rgba(225,235,245,60%), stop: 1 rgba(225,235,245,30%));
}

QToolButton:pressed{
    border: 1px solid rgba(120,160,255,80%);
    background-color: QLinearGradient(spread:pad, x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 rgba(150,180,200,50%),stop: 0.5 rgba(225,235,245,80%), stop: 1 rgba(225,235,245,50%));
}

QToolButton:checked{
    border: 1px solid rgba(120,160,255,40%);
    background-color: rgba(0,0,0,10%)
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset>
       <normaloff>home.ico</normaloff>home.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QToolButton" name="load">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Load stock data file (CSV)</string>
      </property>
      <property name="text">
       <string>Load Data</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>load.png</normaloff>load.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="lasso_btn">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Perform Lasso Regression on the data</string>
      </property>
      <property name="text">
       <string>Lasso</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>lasso.png</normaloff>lasso.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="lin_reg_btn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Perform Linear Regression on the data</string>
      </property>
      <property name="text">
       <string>Linear</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>lin.png</normaloff>lin.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="remove">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Clear the populated data from the table</string>
      </property>
      <property name="text">
       <string>Remove Data</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>remove.png</normaloff>remove.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>20</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>20</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(127, 127, 127);</string>
      </property>
      <property name="text">
       <string>Stock Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(127, 127, 127);</string>
      </property>
      <property name="text">
       <string>Models</string>
      </property>
     </widget>
     <widget class="QToolButton" name="knn_btn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Use K-Nearest Neighbors on the data</string>
      </property>
      <property name="text">
       <string>knn</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>knn.webp</normaloff>knn.webp</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="svm_btn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Use Support Vector Machine on the data</string>
      </property>
      <property name="text">
       <string>svm</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>svm.png</normaloff>svm.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QLabel" name="acc">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="statusTip">
       <string>Acccuracy score of the selected ML model</string>
      </property>
      <property name="text">
       <string>00.00%</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>20</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(127, 127, 127);</string>
      </property>
      <property name="text">
       <string>Accuracy</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset>
       <normaloff>help.png</normaloff>help.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Help</string>
     </attribute>
     <widget class="QToolButton" name="abt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Brief information about the software</string>
      </property>
      <property name="text">
       <string>About</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>about.ico</normaloff>about.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="man">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Short manual of the software</string>
      </property>
      <property name="text">
       <string>Manual</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>manual.png</normaloff>manual.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="lic">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>License notice of the software</string>
      </property>
      <property name="text">
       <string>License</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>license.webp</normaloff>license.webp</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>20</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_10">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>20</width>
        <height>91</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(127, 127, 127);</string>
      </property>
      <property name="text">
       <string>Information</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(127, 127, 127);</string>
      </property>
      <property name="text">
       <string>How to use</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="data_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>781</width>
      <height>431</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
